autoscheduler/frontend/src/tests/AvailabilityUI.test.tsx

```diff
@@ -200,7 +203,7 @@ describe('Availability UI', () => {
       };
       const endEventProps = {
         button: 0,
-        clientY: 100,
+        clientY: 110,
       };
```

Comment:
There are a few manually calculated eventProps like this, is there a reason this can't use `timeToEvent` along with the rest of them? It seems to break `queryByLabelText` (doesn't find a DOM element matching the query) but since this should be the same as what `timeToEvent` returns I don't see why it should matter

Reply:
I chose to leave some of them because those tests focus more on the actual position of the mouse rather than the time. I think you're right about this one, though, so I converted it to use `timeToEvent`. The reason it was throwing an error had to do with the missing `offset`, so I fixed that. Now, the only test that doesn't use`timeToEvent` is the one that checks for correct behavior when dragging below the bottom, which I think just makes more sense if you see the clientY's directly